lib/buildkite.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Get validate pipeline from Buildkite Graphql API
+function get_verification_pipeline {
+	local repo="$1"
+	local api_url="$2"
+	local token="$3"
+	local queryData='{"query":"{organization(slug:\"tapendium\"){pipelines(first:10,repository:{url:\"'"${repo}"'\"}){count edges{node{name slug}}}}}"}'
```

Comment:
ah bk only expose a gql interface?

Reply:
they've still got full support for the rest API but I couldn't see any option to filter the pipelines by git repo in the rest API so this way there's no need to paginate through a whole bunch of results